yt_dlp/extractor/cda.py

```diff
@@ -169,10 +172,10 @@ def _perform_login(self, username, password):
                 'login': username,
                 'password': password_hash,
             })
-        self.cache.store(self._BEARER_CACHE, username, {
+        self.cache.store(self._BEARER_CACHE, username, jwt_encode_hs256({
```

Comment:
why are we encoding a fake JWT? this doesn't make sense and only makes the code more complex

Reply:
it also breaks compat with already existing cache

Reply:
`jwt_is_expired` needs an encoded value, but I'll move the encode to the usage of `jwt_is_expired` so that current cache isn't broken 👍